examples/ui5-js-app/package.json

```diff
@@ -20,7 +20,7 @@
     "test-selenium": "wdio run e2e-test-config/wdio-selenium-service.conf.js"
   },
   "devDependencies": {
-    "@ui5/cli": "^2.14.10",
+    "@ui5/cli": "^3.1.1",
```

Comment:
ui5 tooling v3 requires at least node version 16. Currently we still need to support node 14 that's why our tests are failing

Reply:
agreed, let's put this on a separate PR to both drop node14 and upgrade to ui5-tooling 3